source/bootcamp_content/cloud-infrastructure.rst

```diff
@@ -0,0 +1,101 @@
+Cloud Infrastructure
+====================
+
+*One* cloud, **two** cloud, *my* cloud, **your** cloud.
+
+What The Cloud is
+-----------------
+
+    [...] a model for enabling ubiquitous, on-demand access to a shared pool of
+    configurable computing resources
+
+.. figure:: /static/cloud_computing.png
+    :width: 65%
+    :align: center
+    :target: https://en.wikipedia.org/wiki/File:Cloud_computing.svg
+
+What it looks like
+------------------
+
+.. figure:: /static/cloud_infra_shapes_and_sizes.png
+    :width: 100%
+    :align: center
+
+Advantages over bare hardware
+-----------------------------
+
+Cloud Infrastructure allows you to abstract out *where* the application is
+running and instead focus on more important tasks:
+
+* Performance Optimizations
+* Simplified Debugging
+* Rapid Development / Deployment
+* Easy Scaling
+
+Private Cloud
+-------------
+
+    Infrastructure you run in-house to provide the advantages of cloud
+    computing.
+
+* Openstack
+* CoreOS
+
+Public Cloud
+------------
+
+    Shared resources provided by a third party for personal or corporate use.
+
+* AWS
+* Azure
+* Compute Engine
+
+Cloud + Configuration Management
+--------------------------------
+
+Many CM tools integrate directly, or have plugins to integrate with, Cloud
+tools.
+
+* Ansible modules
+* Chef Server
+* Puppet modules
+* Hashicorp Atlas
+* CoreOS Tectonic
+* Google Kubernetes
+
+Cattle vs Pets: Advantages
+--------------------------
+
+* Cut a release of your application.
+* Spin up a VM on the cloud.
+* Put your app on that VM and auto-deploy it.
+* Everything upgrades with the click of a button.
+* Relatively simple A/B Testing.
+
+Cattle vs Pets: Disadvantages
+-----------------------------
+
+* Configuration Management is an expensive process.
+* Only suitable for large infrastructures, knowing when to *start* can be tough.
+* You just don't need the complexity Cattle Herding creates.
+
+Virtual Machines vs Container
+-----------------------------
+
+Other Ways to Phrase the Argument:
+
+Hypervisor vs Shared kernel
```

Comment:
Maybe add something like this here? 
![image](https://cloud.githubusercontent.com/assets/3160418/12795326/5b187420-ca6e-11e5-948e-6d792b0d05d1.png)